### PR TITLE
docs(readme): update tech stack and interview process

### PR DESCRIPTION
**Pull request by azamb**
small updates to tech stack list and interview process.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 ### #️⃣ About #paid
 
-[#paid](https://www.hashtagpaid.com?utm_source=github.com&utm_medium=referral&utm_campaign=org-page) is a creator marketing platform that sits at the intersection of brand and creator collaboration. We help direct-to-consumer and Fortune 500 brands like Phillips Hue, Everlywell, Unilever, and Sephora, activate creators to drive growth.
+[#paid](https://www.hashtagpaid.com?utm_source=github.com&utm_medium=referral&utm_campaign=org-page) is a creator marketing platform that sits at the intersection of brand and creator collaboration. We help direct-to-consumer and Fortune 500 brands like Infiniti, TJX, Unilever, and Sephora, activate creators to drive growth.
```
Comment (billxinli) [inline profile/README.md:5]: Screenshot 2024-02-07 at 2 40 31 PM

File: profile/README.md [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 
 As the leader in the creator marketing space, #paid offers end-to-end workflow management, including easily matching brands with creators, approving content, automating payments and legal agreements, and reporting on sales impact and ROI.
 
 ### 💻 Our tech stack
 
-We use a range of technologies to get the job done: JavaScript and Node.js coupled with Fastify, React, Ionic, Tailwind CSS, Capacitor, and other libraries to provide a modern, easy-to-use **Javascript** toolchain. A RESTful API back-end powers our platform services.
+We use a range of technologies to get the job done: JavaScript and Node.js coupled with OpenAPI, React, Next.js, storybook and other libraries to provide a modern, easy-to-use Javascript toolchain. A RESTful API back-end powers our platform services.
```
Comment (marcod1419) [inline profile/README.md:11]:
```suggestion
We use a range of technologies to get the job done: JavaScript and Node.js coupled with OpenAPI, React, Next.js, Storybook and other tools to provide a modern, easy-to-use Javascript toolchain. A RESTful API back-end powers our platform services.
```

File: profile/README.md (hunk 1 above)
Comment (billxinli) [reply profile/README.md:5]: can we use TJX?

Comment (azamb) [reply profile/README.md:5]: @billxinli yep, i believe so. I'm using the list of companies currently mentioned in the JDs [here](https://hashtagpaid.com/people#opportunities).

Comment (billxinli) [reply profile/README.md:5]: sounds good

File: profile/README.md [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
+[#paid](https://www.hashtagpaid.com?utm_source=github.com&utm_medium=referral&utm_campaign=org-page) is a creator marketing platform that sits at the intersection of brand and creator collaboration. We help direct-to-consumer and Fortune 500 brands like Infiniti, TJX, Unilever, and Sephora, activate creators to drive growth.
 
 As the leader in the creator marketing space, #paid offers end-to-end workflow management, including easily matching brands with creators, approving content, automating payments and legal agreements, and reporting on sales impact and ROI.
 
 ### 💻 Our tech stack
 
-We use a range of technologies to get the job done: JavaScript and Node.js coupled with Fastify, React, Ionic, Tailwind CSS, Capacitor, and other libraries to provide a modern, easy-to-use **Javascript** toolchain. A RESTful API back-end powers our platform services.
+We use a range of technologies to get the job done: JavaScript and Node.js coupled with Fastify, OpenAPI, React, Tailwind CSS, Next.js, Storybook and other tools to provide a modern, easy-to-use Javascript toolchain. A RESTful API back-end powers our platform services.
```
Comment (billxinli) [inline profile/README.md:11]:
```suggestion
We use a range of technologies to get the job done: JavaScript and Node.js coupled with Fastify, OpenAPI, React, Tailwind CSS, Next.js, Storybook and other open source tools to provide a modern, easy-to-use Javascript toolchain. A RESTful API back-end powers our platform services.
```

**Review by billxinli — DISMISSED** [review]

**Pull request merged**